iphone/cli/commands/_build.js

```diff
@@ -1020,9 +1027,9 @@ iOSBuilder.prototype.configOptionPPuuid = function configOptionPPuuid(order) {
 			let maxAppId = 0;
 			let pp;
 
-			function prep(a, cert) {
+			function prep(a, certs) {
 				return a.filter(function (p) {
-					if (!p.expired && !p.managed && (!cert || p.certs.indexOf(cert) !== -1)) {
+					if (!p.expired && !p.managed && (!certs || intersection(p.certs, certs).length > 0)) {
```

Comment:
So your idea is to grab all certificates, intersect them with the the one in the provided by the build?

Reply:
yeah, intersection of list of certs assigned to the profile and all certs returned by findCertificates.